internal/commands/loglevel/set.go

```diff
@@ -62,22 +62,24 @@ func updateDebuggingConfiguration(ctx context.Context, logSeverity libsveltosv1a
 		)
 	}
 
-	return updateLogLevelConfiguration(ctx, spec)
+	return updateLogLevelConfiguration(ctx, spec, namespace, clusterName)
 }
 
 // Set displays/changes log verbosity for a given component
 func Set(ctx context.Context, args []string) error {
 	doc := `Usage:
-  sveltosctl log-level set --component=<name> (--info|--debug|--verbose)
+  sveltosctl log-level set --component=<name> --namespace=<namespace> --cluster=<cluster-name> (--info|--debug|--verbose)
```

Comment:
we need the cluster-type as well, which can either be Capi or Sveltos

Capi indicates clusterAPI powered cluster
Sveltos indicates a cluster registered with Sveltos (essentially everything else but ClusterAPI powered cluster).